datafusion/core/src/physical_plan/sorts/sort_preserving_merge.rs

```diff
@@ -128,7 +126,7 @@ impl ExecutionPlan for SortPreservingMergeExec {
     }
 
     fn required_input_ordering(&self) -> Vec<Option<Vec<PhysicalSortRequirement>>> {
-        vec![Some(make_sort_requirements_from_exprs(&self.expr))]
+        vec![Some(PhysicalSortRequirement::exact_from_exprs(&self.expr))]
```

Comment:
I am imagining that when a user upgrades DataFusion they will and see that the type signature changed from `PhysicalSortExpr` to `PhysicalSortRequirement`. Then when they look at `PhysicalSortRequirement` they can find a function to convert the existing `PhysicalSortExpr` to `PhysicalSortRequirement` -- `PhysicalSortRequirement::exact_from_exprs`

`make_sort_requirements_from_exprs` was both not documented as well as lost in the list of functions